src/components/AddressSearch/index.js

```diff
@@ -21,7 +21,29 @@ import variables from '@styles/variables';
 import CONST from '@src/CONST';
 import CurrentLocationButton from './CurrentLocationButton';
 import isCurrentTargetInsideContainer from './isCurrentTargetInsideContainer';
-import isSearchedLocation from './isSearchedLocation';
+
+/**
+ * Check if it is the location by searching for the location
```

Comment:
NAB: Update comment to "Check if the place matches the search by the place name or description."